packages/odyssey-react/src/components/Radio/RadioButton.module.scss

```diff
@@ -67,64 +67,34 @@
   &:invalid,
   &.invalid {
     &:hover:not(:checked) + .label::before {
-      border-color: $color-danger-dark;
+      border-color: var(--ColorBorderInvalidHover);
     }
 
     + .label {
-      color: $text-danger;
+      color: var(--ColorTextInvalid);
 
       &::before {
-        border-color: $color-danger-base;
+        border-color: var(--ColorBorderInvalid);
       }
     }
 
     &:checked {
       + .label {
         &::before {
-          background-color: $color-danger-base;
+          background-color: var(--ColorBackgroundInvalid);
         }
       }
     }
 
     &:focus {
       + .label {
         &::before {
-          @include outline($focus-ring-danger, 2px);
+          @include outline(
+            var(--ColorFocusOutlineInvalid),
+            var(--FocusOutlineWidth)
+          );
 
-          border-color: $color-danger-base;
-        }
-      }
-    }
-
-    &:disabled {
```

Comment:
Are these states accounted for elsewhere?

Reply:
As far as the browser is concerned these states are impossible to enter with pseudo classes https://codepen.io/andrewberg-okta/pen/WNZWzJm

I tend to agree that at no point should an element be simultaneously disabled and invalid.

Reply:
That makes sense - `disabled` elements don't get run through constrain validation.

I'm not sure how they're applying them, but these styles were originally requested by folks working on Admin because they force validation onto disabled elements in their "View-Only" mode for forms.

If we're aligned on not supporting non-spec UI states, I am 💯 on this change. 

Note: we'll want to make sure we do preserve these styles under `:readonly` for inputs that support it.